sdk/examples/client/node_api_core/13_get_receipts_migrated_at.rs

```diff
@@ -1,28 +1,40 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! Returns all stored receipts for a given migration index by calling `GET /api/core/v2/receipts/{migratedAt}`.
+//! Returns all stored receipts for a given migration index by querying the `/api/core/v2/receipts/{migratedAt}` node
+//! endpoint.
 //!
-//! `cargo run --example node_api_core_get_receipts_migrated_at --release -- [NODE URL]`
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --example node_api_core_get_receipts_migrated_at <MILESTONE_INDEX> [NODE URL]
+//! ```
+
+use std::env;
 
 use iota_sdk::client::{Client, Result};
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
+
     // Take the node URL from command line argument or use one from env as default.
-    let node_url = std::env::args().nth(1).unwrap_or_else(|| {
-        // This example uses secrets in environment variables for simplicity which should not be done in production.
-        dotenvy::dotenv().ok();
-        std::env::var("NODE_URL").unwrap()
-    });
+    let node_url = env::args().nth(2).unwrap_or_else(|| env::var("NODE_URL").unwrap());
 
-    // Create a client with that node.
+    // Create a client.
     let client = Client::builder().with_node(&node_url)?.finish().await?;
 
+    // Take the transaction id from the command line, or panic.
+    let milestone_index = env::args()
+        .nth(1)
+        .expect("missing example argument: MILESTONE INDEX")
```

Comment:
Similar to output id, should we get the milestone id from the nodeinfo if not provided?

Reply:
yeah, here I would immediately agree since it can be done only using the core api :+1:

Reply:
changed